src/__tests__/connection/enterprise/__snapshots__/hrd_screen.test.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`HRDScreen Component renders correctly when there is an enterprise domai
   data-__type="hrd_pane"
   data-header={
     <p>
-      Please enter your corporate credentials at [ 'domain.com' ].
+      Please enter your corporate credentials at domain.com.
```

Comment:
I'm assuming it's ok that these snapshots have changed? The new output looks better...

Reply:
These appear to change depending on the version of node that generates the snapshots. I think with Node 10 you get the former version, but newer versions omit the wrapping `[' ']`. The output on the actual screen is the same though. These just snuck in from a previous check-in.